tests/test_stimuli/test_game.py

```diff
@@ -0,0 +1,66 @@
+import os
+import retro
+import numpy as np
+from random import random
+from bids_loader.stimuli.game import replay_bk2
+
+
+def test_replay_bk2():
+    bk2_path = "tests/test_stimuli/Airstriker-Genesis-Level1-000000.bk2"
```

Comment:
I think it would be better to pass the game a a parameter with Aistriker as default, so that we can run it on any integration/ROM (to test reproducibility of game specific things: eg. mario sprite flicker due to optimization).

Reply:
ok, but then we also need to pass in the scenario and the inttype (contrary to what I said earlier CUSTOM_ONLY doesn't work with Airstriker-Genesis. Except if we add a custom integration for it, which could be part of the test.